hca/dss/__init__.py

```diff
@@ -760,14 +759,14 @@ def _write_output_manifest(self):
         fieldnames, source_manifest = self._parse_manifest(self.manifest)
         if 'file_path' not in fieldnames:
             fieldnames.append('file_path')
-        with atomic_write(output, overwrite=True, newline='') as f:
+        if os.path.isfile(output):
```

Comment:
`isfile` returns True for symlinks as well. You may want to think through that case. I don't remember what `open(p,'w')` does 

Also, if you fail half-way through writing the manifest, you don't want the file to be half-written. Ideally, there should be no file or only a temporary file (not ideal). That's why `atomicwrite` may still be needed here.